applications/nrf_desktop/src/modules/hid_forward.c

```diff
@@ -997,6 +1023,12 @@ static void disconnect_peripheral(struct hids_peripheral *per)
 	if (per->cfg_chan_rsp) {
 		submit_forward_error_rsp(per, CONFIG_STATUS_WRITE_FAIL);
 	}
+
+	size_t per_id = per - &peripherals[0];
+
+	peripheral_cache[per_id]++;
+	/* A disconnected peripheral is marked with an even number. */
+	__ASSERT_NO_MSG(!(peripheral_cache[per_id] & 1));
```

Comment:
Convert constant `1` to define (e.g. CONNECTED_PEER_DISTINGUISHER or any other name that could hint what it is used for). Also it would be good to define constant as hex `0x01` as it is mainly used in the bitmask operations

Reply:
Changed to `0x01`, thanks for the tip.
I think comment is more meaningful than constant name here (it's difficult to find meaningful name), so I would prefer to keep it as is (also the code is used only in assertions).